lib/components/mp4muxer/mp4muxer-component.js

```diff
@@ -41,8 +41,8 @@ class Mp4MuxerComponent extends Component {
           ftyp.copy(data, 0)
           moov.copy(data, ftyp.byteLength)
 
-          debug('msl:mp4:ftyp')(ftyp)
-          debug('msl:mp4:moov')(moov)
+          debug('msl:mp4:ftyp')(JSON.stringify(ftyp, null, 2))
+          debug('msl:mp4:moov')(JSON.stringify(moov, null, 2))
```

Comment:
This will not print the box hierarchy, so you won't get all the info.
There is already a print function for the `Box` class, but it uses console.warn.
You could modify that to output a string instead, and then use it in the print function, so

from this:

```javascript
/**
   * Pretty-print an entire box as an element/box hierarchy.
   * @param  {Number} [indent=0] How large an indentation to use for the hierarchy
   * @return {undefined}
   */
print (indent = 0) {
    console.warn(' '.repeat(indent), `[${this.type}] (${this.byteLength})`)
    for (const [key, entry] of this.struct) {
      const element = entry.element
      if (element.print) {
        element.print(indent + 2)
      } else {
        console.warn(' '.repeat(indent + 2), key, '=', element.value, `(${element.byteLength})`)
      }
    }
  }
```

to something like this:

```javascript
/**
   * Pretty-format an entire box as an element/box hierarchy.
   * @param  {Number} [indent=0] How large an indentation to use for the hierarchy
   * @return {undefined}
   */
  format (indent = 0) {
    const lines = [' '.repeat(indent) + `[${this.type}] (${this.byteLength})`]
    for (const [key, entry] of this.struct) {
      const element = entry.element
      if (element.print) {
        lines.push(element.format(indent + 2))
      } else {
        lines.push(' '.repeat(indent + 2) + `${key} = ${element.value} (${element.byteLength})`)
      }
    }
    return lines.join('\n')
  }

  /**
   * Pretty-print an entire box as an element/box hierarchy.
   * @param  {Number} [indent=0] How large an indentation to use for the hierarchy
   * @return {undefined}
   */
  print (indent) {
    console.warn(this.format(indent))
  }
```

then use:
```javascript
debug('msl:mp4:ftyp')(ftyp.format())
debug('msl:mp4:ftyp')(moov.format())
```